.github/workflows/reusable_codeql.yml

```diff
@@ -4,10 +4,10 @@ on:
   workflow_call:
     inputs:
         languages:
-          description: "Language for CodeQL code check. Supported languages are: 'python', 'typescript', 'javascript' 'go', 'java', 'cpp', 'csharp', 'ruby'."
+          description: "Provide a comma seperated list. Supported languages for CodeQL code check. Supported languages are: python, typescript, javascript, go, java, cpp, csharp, ruby."
```

Comment:
nit: "supported languages" is doubled up

Reply:
fixed